server/service/integration_enterprise_test.go

```diff
@@ -192,6 +208,8 @@ func (s *integrationEnterpriseTestSuite) TestTeamSpecs() {
 	user, err := s.ds.UserByEmail(context.Background(), "admin1@example.com")
 	require.NoError(t, err)
 
+	// invalid agent options command-line flag
```

Comment:
That comment doesn't seem to apply to the code below?

Reply:
👀 I have no idea how I managed to add that line, thanks! removed in [de1517d](https://github.com/fleetdm/fleet/pull/9442/commits/de1517db471055d5c1ddea3de2fa66335da1ffe5)